### PR TITLE
Include tasks in task_list view

### PR DESCRIPTION
**Pull request by snewcomer**
Add `include` key to tasks for the task_list view

## References
Fixes #

Progress on: #

### CODE REVIEW

File: lib/code_corps_web/controllers/task_list_controller.ex [hunk 1]
```diff
@@ -28,7 +28,7 @@ defmodule CodeCorpsWeb.TaskListController do
     end
   end
 
-  @preloads [:tasks]
+  @preloads [{:tasks, [:comments, :github_pull_request, :task_skills, :user_task]}]
```
Comment (snewcomer) [inline lib/code_corps_web/controllers/task_list_controller.ex:31]: This is using nested preloads.  I am not sure if we should just add the whole list of task preloads or not.  Was just adding them based on the feedback from the tests.  @begedin thoughts?

Comment (snewcomer) [reply lib/code_corps_web/controllers/task_list_controller.ex:31]: ah these are the ones marked with `identifier: always`

Comment (snewcomer) [reply lib/code_corps_web/controllers/task_list_controller.ex:31]: @JoshSmith would you also `include` the task's relationships?

Comment (joshsmith) [reply lib/code_corps_web/controllers/task_list_controller.ex:31]: The real answer here is that it doesn't much matter what I think, but what the performance implications are. Testing this out in the browser with Scout running and doing more precise benchmarking will combined yield a better answer.

**Comment by snewcomer** [comment]
Closing this in favor of #1187

**Pull request closed without merging**